From baa0545371af8f1cd5dfe7606ed7d25db3b22750 Mon Sep 17 00:00:00 2001
From: Yinbo Zhu <yinbo.zhu@nxp.com>
Date: Mon, 8 Apr 2019 19:35:35 +0800
Subject: [PATCH 687/706] arm64: dts: ls1028a: Fix DWC3 IP VBUS glitch issue on
 ls1028a

This patch is to fix DWC3 IP VBUS glitch issue on ls1028a.

Signed-off-by: Yinbo Zhu <yinbo.zhu@nxp.com>
(cherry picked from commit 65ac9c1f71bc0dba016adae20a919251139c7883)
Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
index 9ebe127f8df8..d168b9d4eae1 100755
--- a/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-ls1028a.dtsi
@@ -379,6 +379,7 @@
 			snps,incr-burst-type-adjustment = <1>, <4>, <8>, <16>;
 			usb3-lpm-capable;
 			snps,dis-u1u2-when-u3-quirk;
+			snps,host-vbus-glitches;
 		};
 
 		usb1:usb3@3110000 {
@@ -391,6 +392,7 @@
 			snps,incr-burst-type-adjustment = <1>, <4>, <8>, <16>;
 			usb3-lpm-capable;
 			snps,dis-u1u2-when-u3-quirk;
+			snps,host-vbus-glitches;
 		};
 
 		i2c0: i2c@2000000 {
-- 
2.17.1

